### PR TITLE
Added support for Ubuntu

### PR DESCRIPTION
**Pull request by prahastakumar**
Added support for Ubuntu

### CODE REVIEW

**Comment by eyalneumann** [comment]
LGTM, you mostly just split it for 2 perquisite files, i assume it works,  but someone else from ops should approve as well.

File: defaults/main.yml [hunk 1]
```diff
@@ -18,7 +18,7 @@ __server_port: 8085
 
 __java_version: 17.0.3_7
 
-__nginx_version: 1.19.8
+__nginx_version: 1.24.0
```
Comment (OmprakashPaliwal) [inline defaults/main.yml:21]: Is this required?

File: tasks/main.yml [hunk 2]
```diff
@@ -1,8 +1,16 @@
 ---
 
+- name: Fail if os family is not redhat or debian or windows
+  fail:
+    msg: Current role does not support on {{ ansible_os_family | lower }}, It only supports CentOS/RedHat, Debian and Windows hosts.
+  when: (ansible_os_family | lower) not in ['redhat', 'debian', 'windows']
```
Comment (OmprakashPaliwal) [inline tasks/main.yml:6]: Do we support windows for regional proxy? I guess this is long ago we stopped running on window.

File: tasks/debian-prerequisite.yml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    - name: Add repo
+      apt_repository:
+        repo: "deb http://nginx.org/packages/ubuntu {{ ansible_distribution_release }} nginx"
+        state: present
+        filename: nginx
+      become: yes
+
+    - name: "make sure nginx {{ nginx_version }} is installed"
+      apt:
+        name: "nginx={{ nginx_version }}-1~jammy"
```
Comment (OmprakashPaliwal) [inline tasks/debian-prerequisite.yml:42]: `-1~jammy`. This will not work if we upgrade ubuntu version.

**Review by OmprakashPaliwal — CHANGES_REQUESTED** [review]

File: defaults/main.yml (hunk 1 above)
Comment (prahastakumar) [reply defaults/main.yml:21]: yes, this version is tested and available on both CentOS and Ubuntu

File: tasks/main.yml (hunk 2 above)
Comment (prahastakumar) [reply tasks/main.yml:6]: As far as I know, few customers still use Windows.

File: tasks/debian-prerequisite.yml (hunk 3 above)
Comment (prahastakumar) [reply tasks/debian-prerequisite.yml:42]: yes, we are only supporting Ubuntu 22.04 Jammy for now. If we upgrade Ubuntu, we will have to update the role.

**Review by OmprakashPaliwal — APPROVED** [review]

**Pull request merged**